Azure/azure-sdk-for-media-services
# Add telemetry support for AMS SDK (2nd part).

- add get channel metrics API
- add get streaming endpoints metrics API

E2E test: done

Verification Steps:
- create azure storage table
- insert test data
- perform query and verify the results
